src/v2/providers/alerts/appDistribution.ts

```diff
@@ -45,6 +45,36 @@ export interface NewTesterDevicePayload {
   testerDeviceIdentifier: string;
 }
 
+/**
+ * The internal payload object for receiving in-app feedback from a tester.
+ * Payload is wrapped inside a `FirebaseAlertData` object.
+ */
+export interface InAppFeedbackPayload {
+  ['@type']: 'type.googleapis.com/google.events.firebase.firebasealerts.v1.AppDistroInAppFeedbackPayload';
+  /** Resource name. Format: `projects/{project_number}/apps/{app_id}/releases/{release_id}/feedbackReports/{feedback_id}` */
+  feedbackReport: string;
+  /** Name of the tester */
+  testerName?: string;
+  /** Email of the tester */
+  testerEmail: string;
+  /**
+   * Display version of the release. For an Android release, the display version
+   * is the `versionName`. For an iOS release, the display version is the
+   * `CFBundleShortVersionString`.
+   */
+  displayVersion: string;
+  /**
+   * Build version of the release. For an Android release, the build version
+   * is the `versionCode`. For an iOS release, the build version is the
+   * `CFBundleVersion`.
+   */
+  buildVersion: string;
+  /** Text entered by the tester */
+  text: string;
+  /** URIs to download screenshot(s) */
+  screenshotUris?: string[];
```

Comment:
Can we make this non-optional and always provide the empty array if necessary?

Reply:
In the SDK, correct?

Reply:
Hi there, I'm on Kai's team and working on this while he's on vacation.  

https://github.com/firebase/firebase-functions/pull/1187 includes a fix for this feedback.

Reply:
No worries. I helped out in that PR.